srcs/config/ConfigServer.h

```diff
@@ -35,10 +39,11 @@ class ConfigServer {
   [[nodiscard]] routes_t::const_iterator matchRoute(std::string& path) const;
 
  private:
-  std::vector<std::string> server_name_;
-  uint16_t port_;
-  std::size_t client_max_body_size_;
-  routes_t routes_;
-  std::map<int, std::string> error_pages_;
-  index_files_t files_;
+  routes_t routes_{};
+  std::map<int, std::string> error_pages_{};
+  index_files_t files_{};
+  std::vector<std::string> server_name_{"localhost"};
```

Comment:
You're initializing everything twice now by defining the constructor and the initial values